extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -1773,4 +1773,60 @@ describes.realWin('amp-analytics', {
       expect(getAnalyticsTag(getConfig()).getPriority()).to.equal(1);
     });
   });
+
+  describe('inabox nested transport iframe', () => {
+    let ampMode;
+    beforeEach(() => {
+      ampMode = env.win.AMP_MODE
```

Comment:
Nit: missing semicolon. Maybe also change variable name to indicate it's saving the old value.

Reply:
done